app/uk/gov/hmrc/vulnerabilities/model/TimelineEvent.scala

```diff
@@ -43,3 +43,37 @@ object ServiceVulnerability {
     )(apply, unlift(unapply))
   }
 }
+
+case class VulnerabilitiesTimelineCount(
+  weekBeginning       : Instant,
+  actionRequired      : Int,
+  investigationOngoing: Int,
+  noActionRequired    : Int,
+  uncurated           : Int,
+  total               : Int
```

Comment:
CLARIFICATION: do we still need the counts keyed out per curation status? won't the filter deal with that?

Reply:
resolved :)